actors/builtin/miner/miner_actor.go

```diff
@@ -1507,6 +1516,7 @@ func requestTerminateDeals(rt Runtime, dealIDs []abi.DealID) {
 func requestTerminateAllDeals(rt Runtime, st *State) {
 	// TODO: red flag this is an ~unbounded computation.
 	// Transform into an idempotent partial computation that can be progressed on each invocation.
+	// https://github.com/filecoin-project/specs-actors/issues/411
```

Comment:
This one is the same as the comment above, and distinct from the power/fee calculation issue.